crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -37,7 +37,9 @@
 //! [`EntityWorldMut::remove`]: crate::world::EntityWorldMut::remove
 mod map_entities;
 #[cfg(feature = "bevy_reflect")]
-use bevy_reflect::{Reflect, ReflectDeserialize, ReflectSerialize};
+use bevy_reflect::Reflect;
+#[cfg(all(feature = "bevy_reflect", feature = "serde"))]
+use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
```

Comment:
Can we use prelude here? This looks more cleaner IMO
```suggestion
use bevy_reflect::prelude::*;
```

Reply:
Wildcard imports, particularly in libraries, are, in my opinion, not something we should be doing. They're not as bad as they are in other languages, but they still have a sizable number of issues in library code that I'm not comfortable with.

See https://rust-lang.github.io/rust-clippy/master/index.html#/wildcard_imports.